universal-application-tool-0.0.1/conf/application.conf

```diff
@@ -57,6 +57,7 @@ adfs.secret = ${?ADFS_SECRET}
 adfs.discovery_uri = "https://sts.seattle.gov/adfs/.well-known/openid-configuration"
 adfs.discovery_uri = ${?ADFS_DISCOVERY_URI}
 adfs.admin_group = "ad\\ITD_CiviForm_Admins_Test"
```

Comment:
question (blocking): do we still need this line? Is it a fallback?

Reply:
Yep, for dev, to avoid plumbing the environment variable through out of total laziness.